lib/api/reviews/client.ts

```
@@ -124,22 +124,75 @@ export function useReviewClient() {
+      comment: data.comment
     };
     
-    return transformedReview;
+    try {
+      // First try with the simplified data
+      console.log('Attempting with formatted data:', formattedData);
+      let response;
+      
+      try {
+        response = await apiClient.post<ApiReview>(REVIEW_API.ADD(providerId), formattedData);
+      } catch (initialError) {
```

ESLint

[error] 'initialError' is defined but never used.

(@typescript-eslint/no-unused-vars)